pkg/logs/logs.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2017 Datadog, Inc.
 
-package main
+package logs
```

Comment:
Ultimately your call, but I'd leave this under `cmd`. Think about `pkg` as something you could import from foreign projects to implement new logic - `Start()` is not something I'd import&use, I'd rather use what's in `pkg` to build my own forwarder.